pyart/graph/radarmapdisplay_cartopy.py

```diff
@@ -0,0 +1,510 @@
+            the colorbar.
+        ticks : array
+            Colorbar custom tick label locations.
+        ticklabs : array
+            Colorbar custom tick labels.
+        colorbar_label : str
+            Colorbar label, None will use a default label generated from the
+            field information.
+        ax : Axis
+            Axis to plot on. If provided, must have a Cartopy crs projection.
+        fig : Figure
+            Figure to add the colorbar to. None will use the current figure.
+        lat_lines, lon_lines : array or None
+            Locations at which to draw latitude and longitude lines.
+            None will use default values which are resonable for maps of
+            North America.
+        projection : cartopy.crs class
+            Map projection supported by cartopy.
+        area_thresh : float
+            Coastline or lake with an area smaller than area_thresh in
+            km^2 will not be plotted.
+        min_lat, max_lat, min_lon, max_lon : float
+            Latitude and longitude ranges for the map projection region in
+            degrees.
+        width, height : float
+            Width and height of map domain in meters.
+            Only this set of parameters or the previous set of parameters
+            (min_lat, max_lat, min_lon, max_lon) should be specified.
+            If neither set is specified then the map domain will be determined
+            from the extend of the radar gate locations.
+        lon_0, lat_0 : float
+            Center of the map domain in degrees.  If the default, None is used
+            the latitude and longitude of the radar will be used.
+        shapefile : str
+            Filename for a ESRI shapefile as background (untested).
+        resolution : '10m', '50m', '110m'.
+            Resolution of NaturalEarthFeatures to use. See Cartopy
+            documentation for details.
+        gatefilter : GateFilter
+            GateFilter instance. None will result in no gatefilter mask being
+            applied to data.
+        filter_transitions : bool
+            True to remove rays where the antenna was in transition between
+            sweeps from the plot.  False will include these rays in the plot.
+            No rays are filtered when the antenna_transition attribute of the
+            underlying radar is not present.
+        edges : bool
+            True will interpolate and extrapolate the gate edges from the
+            range, azimuth and elevations in the radar, treating these
+            as specifying the center of each gate.  False treats these
+            coordinates themselved as the gate edges, resulting in a plot
+            in which the last gate in each ray and the entire last ray are not
+            not plotted.
+        embelish: bool
+            True by default. Set to false to supress drawing of coastlines
+            etc.. Use for speedup when specifying shapefiles.
```

Comment:
This should be worded to indicate that the embelishment is only added when the projection allows it.